src/stan/io/deserializer.hpp

```diff
@@ -1229,6 +1254,17 @@ class deserializer {
     return stan::math::sum_to_zero_free(this->read<Ret>(size));
   }
 
+  /**
+   * Read a serialized sum_to_zero matrix and unconstrain it
```

Comment:
Sentences need to end with periods.  In other doc, you've ended parameter descriptions with period.  This we've been very inconsistent about.